system/datatypes/peer_reviewed_article.php

```diff
@@ -1462,6 +1463,17 @@ protected function createCommentDomElement(HyphaDomElement $comment, $review = f
 			return '<span><a href="' . $url . '" target="_blank">' . $url . '</a>' . $suffix.'</span>';
 		}, $commentHtml);
 		$commentHtml = nl2br($commentHtml);
+		if(strlen($commentHtml) > self::MAX_COMMENT_LENGTH){
+			$id = rand();
+			$stub = substr(strip_tags($commentHtml),0, self::MAX_COMMENT_LENGTH*0.7);
+			$commentHtml = '<div style="display:none;" id="commentFull'.$id.'">'.$commentHtml.'</div>
+			<span id="commentStub'.$id.'">'.$stub.'...</span>
+			<br/><a id="commentToggle'.$id.'" onclick="
+				document.getElementById(\'commentStub'.$id.'\').style.display=\'none\';
+				document.getElementById(\'commentFull'.$id.'\').style.display=\'block\';
+				document.getElementById(\'commentToggle'.$id.'\').style.display=\'none\';
```

Comment:
It might be nicer to add some classes and/or `data` attributes here and then put the actual js in a more global spot. OTOH, this is a pragmatic and self-contained approach, of course.

Reply:
Ok, let's leave it like this.